docs/analyse.md

```diff
@@ -40,10 +40,307 @@ watch --interval 30 python -m neps.status ROOT_DIRECTORY
 
 ## Visualizations
 
+### 1. CLI commands
+
 To generate plots to the root directory, run
 
 ```bash
 python -m neps.plot ROOT_DIRECTORY
 ```
 
 Currently, this creates one plot that shows the best error value across the number of evaluations.
+
+### 2. TensorBoard
+
+#### Introduction
+
+[TensorBoard](https://www.tensorflow.org/tensorboard) serves as a valuable tool for visualizing machine learning experiments, offering the ability to observe losses and metrics throughout the model training process. In NePS, we use this powerful tool to show metrics of configurations during training in addition to comparisons to different hyperparameters used in the search for better diagnosis of the model.
+
+#### The Logging Function
+
+The `tblogger.log` function is invoked within the model's training loop to facilitate logging of key metrics.
+
+- **Signature:**
+```python
```

Comment:
Firstly, do you think we can create a dedicated page for this and link it in the `Analyzing` page.

In any case, I feel this section should come up much higher in this page as this is a bigger USP for NePS than whatever is above it. So should be called out and emphasized more.
Also, this can also serve as an HPO search quality status check too! 

So calling it just `visualization` feels heavy underselling.

Reply:
It is now higher in the page to show its importance, and has its own title under `TensorBoard Integration`.